### PR TITLE
Add proxmox action group

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Adds a community.general.proxmox module defaults group (action group).

Ref: #8074

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
meta/runtime.yml
proxmox modules

### CODE REVIEW

File: plugins/doc_fragments/consul.py [hunk 1]
```diff
@@ -56,5 +56,4 @@ class ModuleDocFragment:
     support: full
     membership:
       - community.general.consul
-    version_added: 8.3.0
```
Comment (russoz) [inline plugins/doc_fragments/consul.py:59]: This seems not to be proxmox related.

**Review by russoz — COMMENTED** [review]
I marked one place, but there are many changes in files related to `consul` rather than `proxmox`. Maybe this PRs should be broken in two?

**Comment by felixfontein** [comment]
I moved the consul cleanup to #8350. This PR now only contains proxmox changes.

**Review by russoz — APPROVED** [review]
LGTM

**Comment by felixfontein** [comment]
@russoz thanks again! :)

**Pull request merged**